docs/index.md

```diff
@@ -8,9 +8,9 @@ _cellxgene_ is an interactive data explorer for single-cell transcriptomics data
 #### Single-gene analyses (e.g. expression analysis)
 <img src="customGene.gif" width="600" />
 
-## getting started
+## quick start
 
-_cellxgene_ **only** supports Python 3.6. We recommend [installing _cellxgene_ into a conda or virtual environment.](/faq.html#how-do-i-create-a-python-36-environment-for-cellxgene)
+To install _cellxgene_ you need Python 3.6+. We recommend [installing _cellxgene_ into a conda or virtual environment.](/faq.html#how-do-i-create-a-python-36-environment-for-cellxgene)
```

Comment:
dead link

Reply:
the link should work once the docs compile

Reply:
I think you changed the section heading, though. Will that still work?

Reply:
Um... I think it's the same heading. Am I confused?

Reply:
I think that update goes with this PR actually.

Reply:
perfect. fixed.